packages/atomic/cypress/integration/result-list/folded-result-list-utils.ts

```diff
@@ -0,0 +1,173 @@
+import {
+  generateComponentHTML,
+  ResultWithFolding,
+  SearchFoldedResponseSuccess,
+} from '../../fixtures/test-fixture';
+
+export function withFoldedResult(
+  response: SearchFoldedResponseSuccess
+): SearchFoldedResponseSuccess {
+  response.results[0] = elfCupsAndAllies(opercs, [
+    elfCups(elfCupsAndAllies(), [stubbleLichens(pyrenomycetes)]),
+  ]);
+  return response;
+}
+
+const reusableFields = {
```

Comment:
Yes, I think we can tweak the tests a bit so that instead of looking for specific we just verify that "there's any string".

Also, that source should pretty much be static. It's a push source, so it won't change for no reason over time, it will require someone to repush/rebuild manually.

Reply:
I changed it to request the animals, but I don't think there is a way to guarantee that we are checking that a result with children renders correctly and that a result without children also renders correctly without knowing what the data look like. I though about accessing the `results` array and deciding from that (getting the index of the top level result that matches the criteria, for example), but that seems like a bigger job.
if the source shouldnt change, shall we try to leave it like this and see if it maintains?

Reply:
Yep !

Reply:
You can pick a result out of the one on the org using the `permanentid` field and using the AQ

```
loadAdvancedSearchQueryActions(engine).updateAdvancedSearchQueries({ 
  aq: "@permanentid=5570a7c57da54559b2dde4a1db26c1a9f2b05f277fbb91db49bda76fc8cb",
})
```
<img width="877" alt="Screen Shot 2022-04-05 at 4 37 27 PM" src="https://user-images.githubusercontent.com/4923043/161844822-f30273d4-c22c-4112-9ac5-b2e609ff7bd9.png">

Reply:
you can also filter on the `@foldedcollection`.